wcfsetup/install/files/lib/data/user/User.class.php

```diff
@@ -720,7 +720,7 @@ public function canEmailConfirm()
     }
 
     /**
-     * Returns true, if the user must confirm his email by themself.
+     * Returns true, if the user must confirm his email by himself.
```

Comment:
This change is incorrect, but I believe it should be “themselves” instead of “themself”.

Reply:
Yeah, I wasn't 100% sure about that one.